test/CameraCaptureUITests/wil.natvis

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?> 
```

Comment:
This file is needed? if not, delete.

Reply:
It's not a hard requirement but it does enhance the debugging experience.